src/gsi/setupbend.f90

```diff
@@ -582,23 +582,21 @@ subroutine setupbend(obsLL,odiagLL, &
 
      if(ratio_errors(i) > tiny_r_kind)  then ! obs inside model grid
 
-       if (alt <= six) then
-          if (top_layer_SR >= 1) then ! SR exists for at least one layer. Check if obs is inside 
-             if (tpdpres(i)==ref_rad(top_layer_SR+1)) then !obs inside model SR layer
-                qcfail(i)=.true.
-             elseif (tpdpres(i) < ref_rad(top_layer_SR+1)) then !obs below model close-to-SR layer
-                qcfail(i)=.true.
-             elseif (tpdpres(i) >= ref_rad(top_layer_SR+1) .and. tpdpres(i) <= ref_rad(top_layer_SR+5)) then !source too close
-                qcfail(i)=.true.
-             else !above
-                qcfail(i)=.false.
-                if(hob < top_layer_SR+1) then !correct if obs location is below non-monotonic section
-                   hob = tpdpres(i)
-                   call grdcrd1(hob,ref_rad(top_layer_SR+1),nsig-top_layer_SR-1,1)
-                   data(ihgt,i) = hob+top_layer_SR
-                   hob = hob+top_layer_SR
-                   rdiagbuf(19,i) = hob
-                endif
+       if (alt <= five) then
+          if (top_layer_SR >= 1) then ! SR exists for at least one layer. Check if obs is inside                                                                       
+             if ((tpdpres(i)<ref_rad(top_layer_SR+1)).and.(tpdpres(i)<ref_rad(bot_layer_SR))) then !obs below model SR/close-to-SR layer                                
+                    qcfail(i)=.true.
+             elseif (tpdpres(i)>ref_rad(top_layer_SR+5)) then ! obs above SR/close-to-SR layer                                                                          
+                    qcfail(i)=.false.
+                    if(hob < top_layer_SR+1) then !location might be aliased to the lower section of the non-monotonicity                                                   
+                      hob = tpdpres(i)
+                      call grdcrd1(hob,ref_rad(top_layer_SR+1),nsig-top_layer_SR-1,1) ! only non-monotonic section above SR layer                                           
+                      data(ihgt,i) = hob+top_layer_SR
+                      hob = hob+top_layer_SR
+                      rdiagbuf(19,i) = hob
+                    endif
+             else ! obs inside model SR/shadow or close-to-SR layer                                                                                                     
```

Comment:
Please ensure that the 132 character limit per line is met.  There are several empty spaces on lines 586, 587, 589, 591, 593, and line 598.  After the removal of the blank spaces, line 587 will still exceed the 132 character limit per line.  Please break up line 587 so that the 132 character limit is maintained.